src/constants/Tags.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * This source file is part of the Swift.org open source project
+ *
+ * Copyright (c) 2023-2024 Apple Inc. and the Swift project authors
+ * Licensed under Apache License v2.0 with Runtime Library Exception
+ *
+ * See https://swift.org/LICENSE.txt for license information
+ * See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import { TopicTypes } from 'docc-render/constants/TopicTypes';
+
+export const HIDE_DEPRECATED = 'navigator.tags.hide-deprecated';
+
+export const FILTER_TAGS = {
+  sampleCode: 'filter.tags.sample-code',
+  tutorials: 'filter.tags.tutorials',
+  articles: 'filter.tags.articles',
+  webServiceEndpoints: 'filter.tags.web-service-endpoints',
+};
```

Comment:
`extractTags` is a bit complex logic because there are 3 different types of tags that they don't work in the same way:
- Filter tags: we add them if any symbol has the same type. We should call them "Type tags" instead.
- API Changes tags: we add them when API Changes is turn on and there are tags with the change type
- Deprecated tag: we only add it if there are any deprecated symbols and we don't have API Changes tags

That's why I couldn't really simplify them into a single object, but if you find a way of doing it, please let me know!

Reply:
https://github.com/marinaaisa/swift-docc-render/pull/10

I tried it out and wasn't able to simplify it as much as I initially thought. I added a comment to remind future developers that the categories are also created to keep the suggested tags in a particular order.